src/main/java/uk/gov/hmcts/ccd/data/userprofile/UserProfileLightEntity.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.data.userprofile;
+
+import javax.persistence.*;
+
+/**
+ * This entity differs from UserProfileEntity in a way it has no jurisdictions to speed up performance.
+ */
+@Entity
+@Table(name = "user_profile")
```

Comment:
Do we need a whole new entity type? If the objective is to avoid fetching the expensive 'jurisdictions' relation of UserProfile then we should be able to map the existing UserProfile entity to the new UserProfileLight model.

So long as we don't invoke the getter on the 'jurisdictions' field then the jurisdictions should not be fetched, since hibernate lazy fetches ManyToMany by default.

Reply:
You could potentially use a class based spring projection with your new light model; https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#projections

Reply:
We did'n want to change the Entity behaviour, as other endpoints might be actually fetching the jurisdictions.